data/strings/strings.txt

```diff
@@ -26856,54 +26856,245 @@
+    de = Diese Anwendung benötigt für die Navigation Zugriff auf deinen Standort.
+    el = Αυτή η εφαρμογή απαιτεί πρόσβαση στην τοποθεσία σας για σκοπούς πλοήγησης.
+    es = Esta aplicación necesita acceder a tu ubicación para navegar.
+    et = See rakendus nõuab navigeerimiseks juurdepääsu teie asukohale.
+    fi = Tämä sovellus edellyttää sijaintisi tallentamista navigointia varten.
     fr = L'application a besoin de l'accès à la localisation pour la navigation.
+    hu = Ez az alkalmazás navigációs célokra hozzáférést igényel az Ön tartózkodási helyéhez.
+    id = Aplikasi ini memerlukan akses ke lokasi Anda untuk tujuan navigasi.
+    it = Questa applicazione richiede l'accesso alla tua posizione per la navigazione.
+    ja = このアプリケーションは、ナビゲーションのために位置情報へのアクセスを必要とする。
+    ko = 이 애플리케이션을 사용하려면 내비게이션 목적으로 사용자 위치에 액세스해야 합니다.
+    nb = Denne applikasjonen krever tilgang til posisjonen din for navigasjonsformål.
+    nl = Deze applicatie vereist toegang tot je locatie voor navigatiedoeleinden.
+    pl = Ta aplikacja wymaga dostępu do Twojej lokalizacji do celów nawigacji.
+    pt = Esta aplicação requer o acesso à tua localização para fins de navegação.
+    pt-BR = Este aplicativo requer acesso à sua localização para fins de navegação.
+    ro = Această aplicație necesită accesul la locația dvs. în scopul navigării.
     ru = Этому приложению необходим доступ к вашему местоположению для навигации.
+    sk = Táto aplikácia vyžaduje prístup k vašej polohe na účely navigácie.
+    sv = Denna applikation kräver åtkomst till din plats för navigeringssyften.
+    tr = Bu uygulama, navigasyon amacıyla konumunuza erişim gerektirir.
+    uk = Ця програма потребує доступу до вашого місцезнаходження для навігаційних цілей.
+    zh-Hans = 本应用程序需要访问您的位置以进行导航。
 
   [aa_grant_permissions]
     comment = Displayed on the Android Auto screen. Grant Permissions button. Must be no more than 18 symbols!
     tags = android
     en = Grant Permissions
+    bg = Предоставяне на разрешения
+    cs = Udělení oprávnění
+    da = Giv tilladelser
+    de = Erlaubnisse erteilen
+    el = Χορήγηση αδειών
+    es = Conceder permisos
+    et = Lubade andmine
+    fi = Lupien myöntäminen
     fr = Accorder l'accès
+    hu = Engedélyek megadása
+    id = Memberikan Izin
+    it = Concessione dei permessi
+    ja = 許可を与える
+    ko = 권한 부여
+    nb = Gi tillatelser
+    nl = Toestemming geven
+    pl = Przyznaj uprawnienia
+    pt = Conceder permissões
+    pt-BR = Conceder permissões
+    ro = Acordarea de permisiuni
     ru = Разрешить
+    sk = Udelenie oprávnení
+    sv = Tilldela behörigheter
+    tr = İzinleri Verin
+    uk = Надавати дозволи
+    zh-Hans = 授予权限
 
   [aa_connected_to_car_notification_title]
     comment = Text that will be displayed in the notification title when the app is connected to Android Auto.
     tags = android
     en = Connected to car
+    bg = Свързан с автомобил
```

Comment:
I don't see Belarusian and other languages. Did you use python/tools/translate.py?